sched/scheduler/scheduler_test.go

```diff
@@ -6,31 +6,31 @@ import (
 	"github.com/leanovate/gopter"
 	"github.com/leanovate/gopter/gen"
 	"github.com/leanovate/gopter/prop"
+	"github.com/scootdev/scoot/cloud/cluster/memory"
 	"github.com/scootdev/scoot/saga"
 	"github.com/scootdev/scoot/sched"
-	ci "github.com/scootdev/scoot/sched/clusterimplementations"
-	cm "github.com/scootdev/scoot/sched/clustermembership"
+	"github.com/scootdev/scoot/sched/distributor"
+	"github.com/scootdev/scoot/sched/worker"
 	"testing"
 )
 
 func Test_ScheduleJob_WritingStartSagaFails(t *testing.T) {
 	mockCtrl := gomock.NewController(t)
 	defer mockCtrl.Finish()
 
-	//nodeMock used to ensure no tasks are executed if StartSaga Fails
-	nodeMock := cm.NewMockNode(mockCtrl)
-	nodeMock.EXPECT().Id().Return("node1")
-	cluster, clusterState := cm.DynamicClusterFactory([]cm.Node{nodeMock})
+	nodes := memory.WrapNodes(memory.NewIdNodes(1), worker.NoopWorkerWrapper)
```

Comment:
I would prefer to still use mocks here.  

Once we remove the assumption that everything is running locally, and nodes can fail then we have to write the error handling logic.  Testing this with fakes is difficult.  Testing this with mocks is easy.